app/controllers/katello/api/v2/content_view_filter_rules_controller.rb

```diff
@@ -34,6 +34,7 @@ def index_relation
     param :end_date, String, :desc => N_("erratum: end date (YYYY-MM-DD)")
     param :types, Array, :desc => N_("erratum: types (enhancement, bugfix, security)")
     param :date_type, String, :desc => N_("erratum: search using the 'Issued On' or 'Updated On' column of the errata. Values are 'issued'/'updated'")
+    param :module_stream_ids, String, :desc => N_("module stream ids")
```

Comment:
is this will String or Array similar like errata_ids ?

Reply:
Good catch its array